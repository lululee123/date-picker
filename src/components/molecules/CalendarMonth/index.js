import React from 'react';

import Day from '../../atoms/Day';

import styles from './index.scss';

const CalendarMonth = ({ data = {}, selected, onClick }) => (
	<div className={styles.calendarDay}>
		{Object.keys(data).map(month => (
			<Day
				key={month}
				value={month}
				displayText={month}
				className={styles.month}
				type={data[month] === selected ? 'active' : 'normal'}
				onClick={() => onClick(data[month])}
			/>
		))}
	</div>
);

export default CalendarMonth;
