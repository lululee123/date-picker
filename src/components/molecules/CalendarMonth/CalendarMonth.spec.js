import React from 'react';
import renderer from 'react-test-renderer';

import CalendarMonth from './index';

test('render CalendarMonth correctly', () => {
	const component = renderer
		.create(
			<CalendarMonth />,
		)
		.toJSON();
	expect(component).toMatchSnapshot();
});
