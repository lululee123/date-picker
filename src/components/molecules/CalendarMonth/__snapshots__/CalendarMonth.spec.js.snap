// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CalendarMonth correctly 1`] = `
<div
  className="calendarDay"
/>
`;
